test/specs/basic/ignore-joiners-case-sensitive-diacritics.js

```diff
@@ -0,0 +1,25 @@
+/*!***************************************************
+ * mark.js
+ * https://github.com/julmot/mark.js
+ * Copyright (c) 2014–2016, Julian Motz
+ * Released under the MIT license https://git.io/vwTVl
+ *****************************************************/
+"use strict";
+describe("basic mark with ignore joiners and case sensitive diacritics", function () {
+    var $ctx;
```

Comment:
Just be consistent. It's up to you if you want to write `ignoreJoiners` or `ignore joiners`, but please do it in all these files.